Hymns/Display/DisplayHymnBottomBar.swift

```diff
@@ -118,8 +127,12 @@ struct DisplayHymnBottomBar: View {
                 if !viewModel.songInfo.songInfo.isEmpty {
                     Group {
                         Button(action: {
-                            self.dialogBuilder = {
-                                SongInfoDialog(viewModel: self.viewModel.songInfo).eraseToAnyView()
+                            if self.largeSizeCategories.contains(self.sizeCategory) {
```

Comment:
I saw on the WWDC talk that you can just do `self.sizeCategory >= .extraLarge`. Can you do that instead of defining a list?